.env.sample

```diff
@@ -5,5 +5,6 @@ CFG_VERSION=latest
 CGW_VERSION=latest
 TXS_VERSION=latest
 UI_VERSION=latest
+EVS_VERSION=latest
```

Comment:
I would prefer to use EVENTS_VERSION. For TXS and CGW it makes sense, but I think EVS is really short and a little difficult to understand